From 2b5d8a21aca4e5e3fe744eb2f2009b59eff145e7 Mon Sep 17 00:00:00 2001
From: Kai Pastor <dg0yt@darc.de>
Date: Wed, 17 Mar 2021 07:44:27 +0100
Subject: [PATCH] Don't build tls12-download with MINGW

This fails to build due to missing symbols. And this particular
configuration of tls12-download is not needed because a binary
from a MSVC build is part of the vcpkg-repository.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 24ab128..555c3b1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -190,7 +190,7 @@ endif()
 # === Target: tls12-download ===
 
 set(TLS12_DOWNLOAD_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/tls12-download.c)
-if(WIN32)
+if(WIN32 AND NOT MINGW)
     add_executable(tls12-download ${TLS12_DOWNLOAD_SOURCES})
     set_property(TARGET tls12-download PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded")
     set_property(TARGET tls12-download APPEND PROPERTY LINK_OPTIONS "$<IF:$<CONFIG:Debug>,,/ENTRY:entry>")
-- 
2.17.1

